### ISSUE TITLE
Not working in VSCode integrated terminal 

### ISSUE THREAD

**Issue by Kastakin** [powerline/powerline#2089]
Powerline status works perfectly in my shell while using GNOME-terminal but I can't get it to work on my VSCode integrated Terminal.
Attached there's a screenshot of my .bashrc content relative to the Powerline installation and the error I get in the integrated terminal

![Schermata da 2020-04-16 11-14-45](https://user-images.githubusercontent.com/15200602/79438284-a158cd80-7fd3-11ea-851f-5243d03c4968.png)

**Comment by PH111P** [comment]
It looks like you have a local installation of powerline but you are trying to call `powerline-daemon` as if powerline was installed globally; this should explain the first two lines of the log (and to some extent also the last two lines).

For the last two lines (which cause that you don't see any fancy powerlines), it looks like the file `[...]/powerline-config` does not exist or has the wrong file permissions.

Things you can check/try:
* Does the file `[...]/powerline-config` mentioned in your logs exist and is it readable for the VSCode terminal?
* You can try to symlink your local installation's files to the place where a global installation would install them (something like `ln -s {path_to_powerline}/scripts/powerline ~/.local/bin`; if you've done that already, perhaps VSCode has problems with symlinks

**Comment by Kastakin** [comment]
I'm sorry if it took a bit to report back but I played around with VSCode and I managed to come back with a solution.
At first I tried to follow your troubleshooting intructions with no success, I was about to just ignore the "ugly" prompt-line in the integrated terminal and move on when I discovered that some commands from the Android SKD cmdline weren't recognized by the integrated terminal aswell.
I searched and found in some old Stackoverflow thread abou this last issue issue where it was suggested to change two settings for VSCode:
`"terminal.integrated.shell.linux": "/bin/bash"`
to force the terminal to use the desired shell, and:
`"terminal.integrated.inheritEnv": true`

After a restart both the "command not found" and the prompt not displaying powerline issues were solved.

So turns out it's not a Powerline issue at all and VSCode just need a bit of tweaking in case the prompt is not displayed correctly

Sorry for wasting your time, I hope this solution could be useful to someone else having the same problem as I had since I couldn't find anything about it at the time.

Might be a good idea to add the required configs to the docs in a VSCode section
